lib/ansible/modules/windows/win_share.py

```diff
@@ -84,7 +84,12 @@
       - Specify user list that should get no access, regardless of implied access on share, separated by comma.
     required: no
     default: none
-author: Hans-Joachim Kliemeck (@h0nIg)
+  caching_mode:
+    description:
+      - Set the CachingMode for this share Options: BranchCache, Documents, Manual, None, Programs, Unkown
+    requred: no
+    default: Manual
```

Comment:
CI failure due to the `:` in the description:

```
2017-02-01 18:27:46 ============================================================================
2017-02-01 18:27:46 lib/ansible/modules/windows/win_share.py
2017-02-01 18:27:46 ============================================================================
2017-02-01 18:27:46 ERROR: DOCUMENTATION.options.caching_mode.description.0: expected basestring @ data['options']['caching_mode']['description'][0]. Got None
```

You probably meant this instead?

```yaml
    description:
      - Set the CachingMode for this share.
    choices:
      - BranchCache
      - Documents
      - Manual
      - None
      - Programs
      - Unknown
    required: no
```

Also, note the typos in `Unkown` and `requred` (fixed in my example above).